Doc/library/concurrent.futures.rst

```diff
@@ -252,6 +252,11 @@ to a :class:`ProcessPoolExecutor` will result in deadlock.
    pending jobs will raise a :exc:`~concurrent.futures.process.BrokenProcessPool`,
    as well as any attempt to submit more jobs to the pool.
 
+   *maxtasksperchild* is an optional argument that specifies the maximum
+   number of tasks a single process can execute before it will exit and be
+   replaced with a fresh worker process. The default *maxtasksperchild* is
+   ``None`` which means worker processes will live as long as the pool.
+
    .. versionchanged:: 3.3
```

Comment:
You'll need to add a `versionchanged` clause below to document when the new argument was added.